src/Form/State.elm

```diff
@@ -40,6 +42,16 @@ type alias FormState =
     }
 
 
+{-| Settings for forms initialization:
+
+  - `customFormats` where keys are the accepted custom formats (e.g., personal-number-se\_bank\_id) and values are validation functions for the formats.
+
+-}
+type alias Settings =
+    { customFormats : Dict String (String -> Result String String)
```

Comment:
As we discussed, returning an error may be an overkill. Perhaps return `Result () String`?